compat/src/util.js

```diff
@@ -13,4 +13,4 @@ export function shallowDiffers(a, b) {
 }
 
 export const IS_NON_DIMENSIONAL =
-	/acit|ex(?:s|g|n|p|$)|rph|grid|ows|mnc|ntw|ine[ch]|zoo|^ord|itera/i;
+	/^(-|f[lo].*[^se]$|g.{5,}[^ps]$|z|o[pr]|(W.{5})?[lL]i.*(t|mp)$|an|(bo|s).{4}Im|sca|m.{6}[ds]|ta|c.*[st]$|wido|ini)/;
```

Comment:
My main concern here is that this is a significant byte size increase for something that I'm not sure is worth it